aiida/manage/configuration/profile.py

```diff
@@ -140,72 +112,36 @@ def uuid(self):
             return None
 
     @uuid.setter
-    def uuid(self, value):
+    def uuid(self, value: Optional[str]):
```

Comment:
Do we want this to be optional? I realize that is what it currently it is for this setter, but maybe we should correct this? What is relying on allowing this to be set to `None`?